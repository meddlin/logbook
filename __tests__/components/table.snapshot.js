import { render } from '@testing-library/react';
import Table from '../../components/table';

it('renders Table unchanged', () => {
    const { container } = render(<Table />);

  expect(container).toMatchSnapshot()
})