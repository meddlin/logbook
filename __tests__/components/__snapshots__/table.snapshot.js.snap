// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Table unchanged 1`] = `
<div>
  <div>
    <table>
      <thead>
        <tr>
          <th>
            firstName
          </th>
          <th>
            <span>
              Last Name
            </span>
          </th>
          <th>
            Age
          </th>
          <th>
            <span>
              Visits
            </span>
          </th>
          <th>
            Status
          </th>
          <th>
            Profile Progress
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            tanner
          </td>
          <td>
            <i>
              linsley
            </i>
          </td>
          <td>
            24
          </td>
          <td>
            100
          </td>
          <td>
            In Relationship
          </td>
          <td>
            50
          </td>
        </tr>
        <tr>
          <td>
            tandy
          </td>
          <td>
            <i>
              miller
            </i>
          </td>
          <td>
            40
          </td>
          <td>
            40
          </td>
          <td>
            Single
          </td>
          <td>
            80
          </td>
        </tr>
        <tr>
          <td>
            joe
          </td>
          <td>
            <i>
              dirte
            </i>
          </td>
          <td>
            45
          </td>
          <td>
            20
          </td>
          <td>
            Complicated
          </td>
          <td>
            10
          </td>
        </tr>
      </tbody>
      <tfoot>
        <tr>
          <th>
            firstName
          </th>
          <th>
            lastName
          </th>
          <th>
            age
          </th>
          <th>
            visits
          </th>
          <th>
            status
          </th>
          <th>
            progress
          </th>
        </tr>
      </tfoot>
    </table>
  </div>
</div>
`;
